src/server/JasmineGraphInstanceService.cpp

```diff
@@ -931,7 +939,7 @@ void JasmineGraphInstanceService::trainPartition(string trainData) {
 
     std::string path = "cd " + Utils::getJasmineGraphProperty("org.jasminegraph.graphsage") + " && ";
     std::string command =
-        path + "python3.8 -m unsupervised_train >  /home/ubuntu/software/jasminegraph/logs/unsupervised_train" +
+        path + "python3 -m unsupervised_train >  /home/ubuntu/software/jasminegraph/logs/unsupervised_train" +
```

Comment:
Lets change the path where the log file getting created from `/home/ubuntu/software/jasminegraph/logs` to `/var/tmp/jasminegraph/logs`. Latter is the standard location we maintain runtime logs.

Reply:
Done